Auth: Docs add what needs to happen to change `oauth_skip_org_role_update_sync` (#63203)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

> **Note**: This option is deprecated in favor of OAuth provider specific `skip_org_role_sync` settings. The following sections explain settings for each provider.

> If you wish to change the `oauth_skip_org_role_update_sync` setting to `false`, then for each provider that you have set up, specify whether you want to skip the synchronization or not by using the `skip_org_role_sync` setting.

Reviewer: ```suggestion
If you want to change the `oauth_skip_org_role_update_sync` setting to `false`, then for each provider you have set up, use the `skip_org_role_sync` setting to specify whether you want to skip the synchronization.
```
